exporters/autoexport/README.md

```diff
@@ -0,0 +1,56 @@
+# Automatic Exporter configuration
+
+This module provides easy access to configuring a trace exporter that can be used when configuring an OpenTelemetry Go SDK trace export pipeline.
+
+The autoexport package looks for the `OTEL_TRACES_EXPORTER` environment variable and if set, attempts to load the exporter from it's registry of exporters. The registry is always loaded with an OTLP exporter with the key `"otlp"` and additional exporters can be registered using `autoexport.RegisterSpanExporter("name", ...)`(See example below). Exporter registration uses a factory pattern to not unneccarily build exporters and use resources until they are requested.
+
+If the environment variable is not set, the fallback exporter is returned. The fallback exporter defaults to an [OTLP exporter](https://github.com/open-telemetry/opentelemetry-go/tree/main/exporters/otlp/otlptrace) and can be overriden using the `WithFallbackSpanExporter` option.
```

Comment:
The hyperlinks should direct to https://pkg.go.dev/ (not GitHub)

Reply:
Moreover the `WithFallbackSpanExporter` function should have a hyperlink to https://pkg.go.dev/